### PR TITLE
[Adaptive Card-based Loop Components public preview content]

### PR DESCRIPTION
**Pull request by mobajemu-msft**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/loop-component.md](https://github.com/MicrosoftDocs/msteams-docs/blob/59169991c0a7bce4946009cbbf9b252e2d396d25/msteams-platform%2Fm365-apps%2Floop-component.md) | 87 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/acbe70bf-ef53-415f-a1ea-97059507f761) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/loop-ux-guide.md](https://github.com/MicrosoftDocs/msteams-docs/blob/59169991c0a7bce4946009cbbf9b252e2d396d25/msteams-platform%2Fm365-apps%2Floop-ux-guide.md) | 87 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bda19bc4-31bc-4d6a-8edd-2df877fb219e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/loop-component.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6caf12ffb1f7dbde352e4156277e2f53c3a401c8/msteams-platform%2Fm365-apps%2Floop-component.md) | 87 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/582c8f28-add1-4844-b268-4028173e4ac8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/loop-ux-guide.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6caf12ffb1f7dbde352e4156277e2f53c3a401c8/msteams-platform%2Fm365-apps%2Floop-ux-guide.md) | 87 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e9828456-471d-43c9-a6fe-9b738c5637c7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/loop-component.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3986303ce680d56c9bc7357c1e4f86f763dc68a0/msteams-platform%2Fm365-apps%2Floop-component.md) | 87 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9be1ae68-3565-45c5-a047-dfc03a6709e5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/loop-ux-guide.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3986303ce680d56c9bc7357c1e4f86f763dc68a0/msteams-platform%2Fm365-apps%2Floop-ux-guide.md) | 87 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ae8b917e-2b21-4dff-8477-3459244c5e37) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by erikadoyle — COMMENTED** [review]
@mobajemu-msft for TOC placement, could you add a subnode **Adaptive Card Loop components** to "Extend your app across M365", right between ME and publish nodes (the green here):

image

. . . also don't forget to toggle off *draft* mode to signal this is ready for review. Thanks!

File: msteams-platform/m365-apps/loop-component.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ms.localizationpriority: medium
+---
+
+# Guidelines for building Loop components 
+
+## Overview
+
+Loop component is an evolution of Fluid component - a way to collaborate with a team through loop whether it’s in chat, email, meeting, or Loop page. Loop component stays in sync no matter how many places the content lives across Microsoft 365 apps.
+
+## Getting started
```
Comment (shuchauh) [inline msteams-platform/m365-apps/loop-component.md:17]: Add a note before the getting started section:

[!NOTE]  
Adaptive Card-based Loop components are available in public preview.

**Review by shuchauh — CHANGES_REQUESTED** [review]
Kindly incorporate the proposed changes by 8/14. Feel free to reach out for any clarification over the Teams chat.

Thanks!

File: msteams-platform/m365-apps/loop-component.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+To turn on the Adaptive Card based Loop component in Outlook.com:
+
+1. [Launch Outlook Web with feature flight override](https://outlook-sdf.office.com/mail/?&cardLoop.componentEnabled=on)
+1. Login with the test tenant credentials shared with you by Microsoft
+1. Click on the rollout override button in the top ribbon
+1. Make sure to **turn on** these flights - **cmp-fluid-devOnlyGenericLoaderAllLinks, cmp-fluid-chapterThree** and to 
+turn off this flight, **cmp-fluid-useProvidersForME**
+    > [!NOTE]
+    > The above flags should be turned on/ off at both sender’s and receiver’s end for the experiences to work seamlessly.
```
[the remark this answers is not in the document]
Comment (mobajemu-msft) [reply msteams-platform/m365-apps/loop-component.md:78]: is this recommending the complete removal of the 'setup your dev environment to test in outlook' or replacing it?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3bcdaf87392e4f8f7b8a24ae189267da0ed2e8a5/msteams-platform%2FTOC.yml) | 99 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4fc3b382-59f8-435c-96de-fb273a1b622b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/loop-component.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3bcdaf87392e4f8f7b8a24ae189267da0ed2e8a5/msteams-platform%2Fm365-apps%2Floop-component.md) | 75 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aa93c50e-0150-4157-b12d-f539b2c273ff) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/loop-ux-guide.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3bcdaf87392e4f8f7b8a24ae189267da0ed2e8a5/msteams-platform%2Fm365-apps%2Floop-ux-guide.md) | 87 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c050edb3-2210-4d3d-9a84-98b569bc7206) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/loop-component.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+ms.custom: m365apps
+ms.localizationpriority: medium
+---
+
+# Guidelines for building Loop components 
+
+## Overview
+
+Loop component is an evolution of Fluid component - a way to collaborate with a team through loop whether it’s in chat, email, meeting, or Loop page. Loop component stays in sync no matter how many places the content lives across Microsoft 365 apps.
+
```
[the remark this answers is not in the document]
Comment (mobajemu-msft) [reply msteams-platform/m365-apps/loop-component.md:16]: @shuchauh I cut the second sentence of the first paragraph from my latest commit. The phrase 'Actionable units of productivity' sounds very jargony and most likely will go over the head of most readers. The portion about staying in sync across M365 was captured already in the second paragraph. 

I could be wrong about these changes, awaiting your feedback.

**Comment by v-ypalikila** [comment]
Incorporated all the updates in the PR  in #9230 .
Closing this PR.

**Pull request closed without merging**